bugshaker/src/main/java/com/github/stkent/bugshaker/utilities/LogcatUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.github.stkent.bugshaker.utilities;
+
+import java.io.File;
+import java.io.IOException;
+
+import android.content.Context;
+
+/**
+ * Created by aschen on 7/20/16.
+ */
```

Comment:
We should remove this comment. Maybe look up a way on AndroidStudio to prevent creating a comment when you create a new file.